ee/src/main/java/org/jboss/as/ee/component/DefaultComponentViewConfigurator.java

```diff
@@ -78,6 +78,14 @@ public void configure(final DeploymentPhaseContext context, final ComponentDescr
                 }
             }
 
+            Class<?> markupClass;
+            try {
+                markupClass = module.getClassLoader().loadClass(view.getMarkupClassName());
```

Comment:
This should handle the possibility that view.getMarkupClassName() will return null.